cpp/include/cudf/strings/regex/flags.hpp

```diff
@@ -33,9 +33,10 @@ namespace strings {
  * and to match the Python flag values.
  */
 enum regex_flags : uint32_t {
-  DEFAULT   = 0,  ///< default
-  MULTILINE = 8,  ///< the '^' and '$' honor new-line characters
-  DOTALL    = 16  ///< the '.' matching includes new-line characters
+  DEFAULT   = 0,   ///< default
+  MULTILINE = 8,   ///< the '^' and '$' honor new-line characters
+  DOTALL    = 16,  ///< the '.' matching includes new-line characters
+  ASCII     = 256  ///< use only ASCII when matching built-in character classes
```

Comment:
I'm curious, why the big jump? Do we anticipate sticking other flags in between `DOTALL` and `ASCII`? For that matter, there seems to also be a jump from `DEFAULT` to `MULTILINE`, not sure if that's significant.

Reply:
As mentioned in the doxygen for `regex_flags` here in this file, these are set to match the Python flag values.
This makes it a bit easier to call through Cython so we don't have to map any values.
```
>>> import re
>>> int(re.MULTILINE)
8
>>> int(re.DOTALL)
16
>>> int(re.ASCII)
256
>>> 
```

Reply:
Ah thanks I didn't realize that.